Jellyfin.Api/Controllers/SessionController.cs

```diff
@@ -436,9 +441,9 @@ public class SessionController : BaseJellyfinApiController
         [ProducesResponseType(StatusCodes.Status204NoContent)]
         public ActionResult ReportViewing(
             [FromQuery] string? sessionId,
-            [FromQuery] string? itemId)
+            [FromQuery, Required] string? itemId)
         {
-            string session = RequestHelpers.GetSession(_sessionManager, _authContext, Request).Id;
+            string session = sessionId ?? RequestHelpers.GetSession(_sessionManager, _authContext, Request).Id;
```

Comment:
The old behaviour is to overwrite the sessionId no matter what. It doesn't matter though. This endpoint is entirely unused.

Reply:
if that's the case, are we better removing sessionId altogether?

Reply:
Afaik, we're not making any breaking changes to the API in 10.7, but I could be wrong.